### DISCUSSION TITLE
Adding Elestio as deployment option

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [mealie-recipes/mealie#4377] [Ideas]
Hey team,
I am Kaiwalya, Developer Advocate at Elestio. Elestio has been providing options of fully deploying and managing Mealie application as shown [here](https://elest.io/open-source/mealie). I think it would be a great idea if we can add it to official readme/documentation here.
🔔 In addition to this, if you are interested we provide partnership opportunities with tools we support by **revenue share** upon addition of this method in docs. If you would like to collaborate, just shoot me an email at [kaiwalya@elest.io](mailto:kaiwalya@elest.io) :)

**Comment by Kuchenpirat** [comment]
Thanks for reaching out. 
We will discuss the matter internally. As this is not an mealie issue i will convert this to a discussion.
